pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go

```diff
@@ -231,6 +231,15 @@ type PGBackRestRepoHost struct {
 	// +optional
 	TopologySpreadConstraints []corev1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
 
+	// Custom sidecar containers for the pgBackRest repo host pod. Changing this
+	// value causes the repo host to restart.
+	// +optional
+	Containers []corev1.Container `json:"containers,omitempty"`
+
+	// Environment variables to be set in the pgBackRest repo host container.
+	// +optional
+	Env []corev1.EnvVar `json:"env,omitempty"`
```

Comment:
Good day, @jsierles. What do you think about using Secret for this need? We have it for our PXC operator https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L472 I think it is better because env vars can have sensitive information and it is not good to set values via CR.

Reply:
Additionally, we can add a separate secret option for each component (not just for the pgBackRest repo host) as we have in PXCO. This could be useful for you in the future as well.

Reply:
That seems fine. In our case we're not setting sensitive information, but we can change it to use secrets.

Reply:
It is ok, we can wait. I just want to be sure that you don't have any questions/blockers. Thank you.